app/renderer/renderer.py

```diff
@@ -13,6 +13,31 @@ def __init__(self, schema, response_store, validation_store, navigator, metadata
         self._navigator = navigator
         self._metadata = metadata
 
+        # Create an object from a dict
+        class ContextEntry:
+            def __init__(self, entries):
+                self.__dict__ = entries
+
+        start_date = None
+        end_date = None
+
+        try:
+            start_date = self._metadata.get_ref_p_start_date()
+            end_date = self._metadata.get_ref_p_end_date()
+        except:
+            pass
```

Comment:
I don't really understand this try/except block. Either catch the exception and log it or at the very least comment whats going on as to me this looks like it will just quietly swallow all errors.